guildwars2/account.py

```diff
@@ -379,6 +390,9 @@ def count_upgrades(slots):
             for item in v:
                 count += get_amount_in_slot(item)
             storage_counts[k] = count
+        storage_counts["tp delivery"] = 0
+        for item in delivery:
```

Comment:
Since you define `delivery` in that try/except block, if it fails on the api calls `delivery` will be undefined here